hardware/raspberrypi/bootmodes/msd.md

```diff
@@ -1,24 +1,50 @@
-* Update to Raspberry Pi OS 2020-08-20 or newer.
-* Use [raspi-config](../../../configuration/raspi-config.md) to choose between SD/USB (default) or SD/Network boot modes.
+To boot from a USB mass storage device, the Raspberry Pi 4B requires the bootloader EEPROM release from 3 September 2020 or later. Check which bootloader EEPROM version your Pi has using the `rpi-eeprom-update` tool:
 
-The full set of boot mode options is documented on the [bootloader configuration](../bcm2711_bootloader_config.md) page.
+```bash
+pi@raspi4b:~ $ sudo rpi-eeprom-update
+BCM2711 detected
+Dedicated VL805 EEPROM detected
+BOOTLOADER: up-to-date
+CURRENT: Thu  3 Sep 12:11:43 UTC 2020 (1599135103)
+ LATEST: Thu  3 Sep 12:11:43 UTC 2020 (1599135103)
+ FW DIR: /lib/firmware/raspberrypi/bootloader/critical
+VL805: up-to-date
+CURRENT: 000138a1
+ LATEST: 000138a1
+ ```
+ 
+The `rpi-eeprom-update` tool reports information about two different EEPROMs: the bootloader EEPROM and the VL805 USB controller EEPROM. The bootloader EEPROM information is reported after the `BOOTLOADER` heading: here the `CURRENT` release is what is installed in this Pi's EEPROM and is the 3 September 2020 release, so this Pi can do USB mass storage boot since it has the required version. The `LATEST` version shown is the most recent version available from the [APT](https://www.raspberrypi.org/documentation/linux/software/apt.md) package present on this Pi.
+ 
+ If your Pi has an earlier release of bootloader EEPROM, there are two ways to upgrade to a version that can do USB mass storage boot:
+ 
+ 1. Use the `Misc Utility Images` option in the [Raspberry Pi Imager](https://www.raspberrypi.org/downloads/) to create an SD card with the latest `Raspberry Pi 4 EEPROM boot recovery` image. Once you have imaged the SD card, boot the Pi with this card and a display connected to one (or both) of the HDMI ports. As the Pi boots, its bootloader EEPROM will be upgraded to the latest release. The display will turn green to indicate a successful update.
+ 1. Run Raspberry Pi OS 2020-08-20 or later from an SD card, and use it to upgrade the EEPROM. First, upgrade the packages on your system using the command `sudo apt update` followed by `sudo apt upgrade`. Once you have done this, run `sudo rpi-eeprom-update`: it will present a `LATEST` version of the bootloader EEPROM from 3 September 2020 or later. Once you can see this new EEPROM version in the output of `rpi-eeprom-update`, reboot the Pi to install it.
+ 
+Once your Pi 4B has the 3 September 2020 or later bootloader EEPROM release, it can boot from a USB mass storage device. If an SD card is present, the Pi will attempt to boot from that first, before attempting boot from USB mass storage.
+
+You can use the `raspi-config` tool to swap between SD/USB boot, and SD/network boot modes. The full set of boot mode options is documented on the [bootloader configuration](../bcm2711_bootloader_config.md) page.
+ 
+<a name="pi400"></a>
```

Comment:
> The new 40 second installation video is gold standard IMHO
https://www.youtube.com/watch?v=J024soVgEeM

Ha, that says "Raspbian" rather than "Raspberry Pi OS" :wink:

Reply:
> > The new 40 second installation video is gold standard IMHO
> > https://www.youtube.com/watch?v=J024soVgEeM
> 
> Ha, that says "Raspbian" rather than "Raspberry Pi OS" 😉

The video itself doesn't mention Raspbian - it's just the title it has been given on YouTube. I'm sure if you poked the right person that would be an easy change to make 😉

Reply:
> For USB boot I think we just need a single place where we can link to from this / other documents which says
> The baseline version of Raspberry Pi OS is X and the Bootloader Version is Y. Use the Raspberry Pi Imager to get to this version.
> e.g. If the HDMI diagnostics screen doesn't show a minimum version of X then upgrade for USB boot support

Agreed - I've added this.